### PR TITLE
New card component

### PR DESCRIPTION
**Pull request by joebudi**
## Description
_Describe the problem or feature in addition to a link to the relevant github issues._
#531 New card component to replace the current material card.

## Screenshots
Vertical
https://www.loom.com/share/76758a7c621948fd9771cdb56820857c

Horizontal
https://www.loom.com/share/2832f2c9ea7040b9bda064a9473fc701

### CODE REVIEW

File: packages/builder/src/components/userInterface/temporaryPanelStructure.js [hunk 1]
```diff
@@ -311,6 +311,174 @@ export default {
...
+                    key: "cardWidth",
+                    control: OptionSelect,
+                    options: ["16rem", "20rem", "24rem"],
+                    placeholder: "Card Width",
+                  },
+                ],
+              },
+            },
+            {
+              _component: "@budibase/standard-components/cardhorizontal",
```
Comment (shogunpurple) [inline packages/builder/src/components/userInterface/temporaryPanelStructure.js:390]: Having separate components for card + horizontal might be overkill. I think we should have a "Card" component, with a prop to set it as vertical/horizontal instead

**Review by shogunpurple — COMMENTED** [review]
Code looks good - but I think we should consolidate the 2 components

File: packages/builder/src/components/userInterface/temporaryPanelStructure.js (hunk 1 above)
Comment (joebudi) [reply packages/builder/src/components/userInterface/temporaryPanelStructure.js:390]: TBH I am not sure how to do this lol

Comment (joebudi) [reply packages/builder/src/components/userInterface/temporaryPanelStructure.js:390]: Also, they have different props - width / height / image size props

Comment (shogunpurple) [reply packages/builder/src/components/userInterface/temporaryPanelStructure.js:390]: cool, it's probably more hassle than it's worth then due to them having different props. I think we need a way to hide certain props based on the values of the others in the builder so we don't have to create a ton of different components

**Review by shogunpurple — APPROVED** [review]

**Pull request merged**